packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -57,6 +57,11 @@ const cli = {
         type: 'array',
         describe: 'A list of spec file paths.',
       },
+      updateScreenshots: {
```

Comment:
How closely do we want to follow jest's cli options? Our screenshots are nested inside of `__snapshots__`. So this could be a 1-1 command `--updateSnapshot --u`. Although for consumers I think `updateScreenshot(s)` might be more intuitive. I've noticed WebDriverIO and jest both use singular option names, I'm not sure why. 

https://jestjs.io/docs/en/cli#--updatesnapshot

I'm not strongly opinionated towards either option, but I wanted to have a small discussion before we settled on the final commit changes.

Reply:
I added the `u` alias for now. We can discuss in our sync to see what everyone thinks about the alias and naming. https://github.com/cerner/terra-toolkit/pull/568/commits/07340103010232d3e8f52accdb9e9f65c44e0371